bitnami/elasticsearch/templates/curator/configmap.yaml

```
@@ -13,9 +13,9 @@ metadata:
   annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
   {{- end }}
 data:
-  action_file.yml: {{ required "A valid .Values.curator.configMaps.action_file_yml entry is required!" (toYaml .Values.curator.configMaps.action_file_yml | indent 2) }}
+  action_file.yml: {{ toYaml (include "common.tplvalues.render" ( dict "value" (required "A valid .Values.curator.configMaps.action_file_yml entry is required!" .Values.curator.configMaps.action_file_yml) "context" $ )) | indent 2 }}
   {{- if .Values.curator.configMaps.config_yml }}
-  config.yml: {{- include "common.tplvalues.render" ( dict "value" .Values.curator.configMaps.config_yml "context" $ ) | nindent 2 }}
+  config.yml: {{- include "common.tplvalues.render" ( dict "value" .Values.curator.configMaps.config_yml "context" $ ) | indent 2 }}
```

Its missing the `toYaml` here:

```suggestion
  config.yml: {{- toYaml (include "common.tplvalues.render" ( dict "value" .Values.curator.configMaps.config_yml "context" $ )) | indent 2 }}
```
```

---

added